lms/static/js/edxnotes/views/search_box.js

```diff
@@ -0,0 +1,154 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'jquery', 'underscore', 'backbone', 'gettext', 'js/edxnotes/utils/logger',
+    'js/edxnotes/collections/notes'
+], function ($, _, Backbone, gettext, Logger, NotesCollection) {
+    var SearchBoxView = Backbone.View.extend({
+        events: {
+            'submit': 'submitHandler'
+        },
+
+        errorMessage: gettext('This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.'),
```

Comment:
@srpearce 
>If this is a user-facing message, I'd suggest changing it to this (for all instances):
An error has occurred. Make sure that you are connected to the Internet, and then try refreshing the page.

Done.